docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+
+While it is definitely possible to get to the same output using a powerful model (and this was actually the source for the instruction dataset), this has some limitations:
+* **Prompts will not systematically work**. While generating the instruction dataset, I’ve had to discard 5% of the generated output. This is not much, but you have to combine this uncertainty with all the others (retrieval failure, imprecise query) and this adds up to the stress of your RAG system.
+* **Prompts can saturate**, especially if on top of asking for an advanced verifiability check, you are also making other recommendations for style (see next section).
+* **Prompts are open to injection and security issues**. This is the infamous “Chevrolet” effect where a help support system turns out to be a thin wrapper of ChatGPT.
+
+Assessing model robustness is the tricky part. Standard benchmarks do not apply: what you are going to measure is actually a loss of performance as the specialization through retraining only results in a loss of general performance.
+
+As far as accuracy goes, the most straightforward way is to compare the model output and the original documents. It works obviously best if you are already clarifying the attribution of individual statements to specific documents through generated references. For this process you can use an LLM-as-a-judge or, if you provide actual quotes, a straight comparison with the original text (as an additional confusing factor, this is called a “text alignment” algorithm, but nothing to do with fine-tuning or RLHF).
+
+### 4.3 Purpose of fine-tuning: relieving data frictions
+A RAG system is made primarily for its users. Hence one of the biggest issues of using a generalist model like GPT-3.5/4 on a long term basis: you have an audience alignment gap. This is even more striking if you’re working in another language than English (like myself): ChatGPT feels immediately “alien” with weird twists of languages that are not to be found in standard French.
+
+Reshaping style is the most shocking impact of fine-tuning. A few months ago I released an extreme version of user-LLM communication: MonadGPT, an instruction model trained on 17th century English. What took me the longest was to come up with the most fitting communication scheme: train the models on a couple of answers from historical texts and generated questions in contemporary English, simply because I should absolutely not expect the users to mimic 17th century language. Users should not have to mimic the style of any document resources either. 
```

Comment:
"shocking" = surprising?

Reply:
So maybe "In my past experiences, fine-tuning is the most effective methods to reshape the style and overall method of communication of an LLM". And then : "train the models" => "fine-tune an LLM".